stdlib/http/src/main/java/org/ballerinalang/net/http/nativeimpl/connection/Respond.java

```diff
@@ -108,14 +113,30 @@ public static Object nativeRespond(ObjectValue connectionObj, ObjectValue outbou
             }
         } catch (ErrorValue e) {
             unBlockStrand(strand);
-            log.debug(e.getPrintableStackTrace(), e);
-            return e;
+            if (log.isDebugEnabled()) {
+                log.debug(e.getPrintableStackTrace(), e);
```

Comment:
Shall we have a meaningful message instead of `e.getPrintableStackTrace()`, since it is there in the `e`

Reply:
These changes have been reverted in #a30ea6fe66758a2e7ed37f42d6754be0cf37e8c5